TestGeneratorSystem/src/testgeneratorsystem/Student.java

```diff
@@ -100,6 +113,69 @@ public double getAverageScore() {
         // Implement logic to calculate average score
         return 0.0; // Placeholder value
     }
+    @Override
+    public void removeAccount(){
+        try {
+           listOfStudents.remove(this); 
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+        System.out.println("the account is removed");
+    }
+    @Override
+    public Student login(){
```

Comment:
remove any system.out.println indicating login success or fail , should be done on the main class